packages/wallets/src/components/WalletsResetMT5Password/WalletSuccessResetMT5Password.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useCallback } from 'react';
+import useDevice from '../../hooks/useDevice';
+import MT5PasswordUpdatedIcon from '../../public/images/ic-mt5-password-updated.svg';
+import MT5SuccessPasswordReset from '../../public/images/mt5-success-password-reset.svg';
+import { ModalStepWrapper, WalletButton } from '../Base';
+import { useModal } from '../ModalProvider';
+import { WalletsActionScreen } from '../WalletsActionScreen';
+
+type WalletSuccessResetMT5PasswordProps = {
+    isInvestorPassword?: boolean;
+    title: string;
+};
+
+const WalletSuccessResetMT5Password: FC<WalletSuccessResetMT5PasswordProps> = ({
+    isInvestorPassword = false,
+    title,
+}) => {
+    const { hide } = useModal();
+    const { isDesktop, isMobile } = useDevice();
+
+    const renderFooter = useCallback(() => {
+        return isMobile ? <WalletButton isFullWidth onClick={() => hide()} size='lg' text='Done' /> : null;
+    }, [isMobile, hide]);
+
+    const renderButtons = useCallback(() => {
+        return isDesktop ? <WalletButton onClick={() => hide()} size='lg' text='Done' /> : <></>;
+    }, [isDesktop, hide]);
```

Comment:
But the types are slightly different for both `RenderFooter` and `RenderButtons` so be aware of it @sergei-deriv

Reply:
@thisyahlen-deriv @aizad-deriv it looks really weird :) In my opinion it will make the code harder to understand.

<img width="1089" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/63cd0297-9808-4d89-8593-b323ff1fb652">

Reply:
Yes that's the thing. The issue is that the design for modals and popups in responsive view are very inconsistent. We just have to make do whatever we have right now. Hopefully they will resolve in the redesign 🙏🏼